packages/components/src/GroupPanel.component.svelte

```diff
@@ -37,22 +38,25 @@
 
   function onClick(namespace, groupID, arrayID) {
     window.dispatchEvent(
-      new CustomEvent(events.BUTTON_CLICK, {
+      new CustomEvent<GUIEvent<ButtonProps>>(Events.BUTTON_CLICK, {
         bubbles: true,
         detail: {
-          id: 'asset_selection',
+          target: 'asset',
           props: {
-            namespace,
-            groupID,
-            arrayID
+            command: 'select',

```

Comment:
I've noticed we use this command pattern is quite a few places, would it make sense to create an enum with all the commands?